Openpype settings project global (#5134)
ynput/OpenPype: website/docs/admin_settings_local.md

## Categories

### OpenPype Mongo URL
The **Mongo URL** is the database URL given by your Studio. More details [here](artist_getting_started.md#mongodb).

### General

> **OpenPype Username** : enter your username (it can also take by default the computer session username). It signs your actions on **OpenPype**.

Reviewer: Admittedly this is unclear to me. How does it sign my actions on OpenPype? What does that influence?

Here is it a bit tweaked. Note that I'm not sure if my example is actually correct and whether it does that.

```suggestion
**OpenPype Username** : enter your username (if not provided, it uses computer session username by default). This username is used to sign your actions on **OpenPype**, for example the "author" on a publish.
```

> **Admin permissions** : on checked, no need to enter a password (if defined) to access to the **Admin** section.

Reviewer: ```suggestion
**Admin permissions** : When enabled you do not need to enter a password (if defined in Studio Settings) to access to the **Admin** section.
```

### Experimental tools

> Futur version of existing tools or new ones.

Reviewer: ```suggestion
Future version of existing tools or new ones.
```

### Environments
Local replacement of the environment data of each software and additional internal data necessary to be loaded correctly.

### Applications

> Local override of software paths and their versions. More details [here](admin_settings_system.md#applications).

Reviewer: ```suggestion
Local override of software executable paths for each version. More details [here](admin_settings_system.md#applications).
```
